docs/src/templates/includes/components/core/je-appsBanner.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="c-appsBanner">
+    <div class="c-appsBanner-image">
```

Comment:
Bit confusing to use `c-appsBanner-image` and `c-appsBanner-img`.  Probably best off calling this one `c-appsBanner-imgWrapper` and keeping the other one as `c-appsBanner-img`

Reply:
Would it be better to just call it `c-appsBanner-wrapper` and `c-appsBanner` (leave of the `img` part) as they're used on images anyway so probably won't need that naming convention?

Reply:
@ashleynolan actually I don't use `c-appsBanner-img` in css so would it be enough to just leave c-appsBanner-image?

Reply:
I've removed `c-appsBanner-img` class, so now it shouldn't be confusing :)